src/pbview.cpp

```diff
@@ -308,6 +321,10 @@ void PbView::run_help()
 			continue;
 		}
 
+		if (strcmp(event, "RESIZE") == 0) {
+			handle_resize();
```

Comment:
Same as in `run()`: shouldn't there be a `continue` here?

Reply:
It is not strictly necessary but it is indeed good to add it for consistency.